products/sle/main.pm

```diff
@@ -1392,6 +1392,10 @@ elsif (get_var('HPC')) {
         }
     }
 }
+elsif (get_var('SYSTEMD_TESTSUITE')) {
+    boot_hdd_image;
+    loadtest 'console/systemd_testsuite';
+}
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html). How about in lib/main_common.pm?

Reply:
I don't understand what you mean. Which code am I repeating?